e2e/test/scenarios/embedding/embedding-smoketests.cy.spec.js

```diff
@@ -89,6 +78,9 @@ describe("scenarios > embedding > smoke tests", { tags: "@OSS" }, () => {
 
       cy.log("Standalone embeds page");
       mainPage().within(() => {
+        cy.findByLabelText("Enable Static embedding")
```

Comment:
toggle the static embedding inside the settings page instead to see if other inputs would update accordingly.